app/src/main/res/values-fr/strings.xml

```diff
@@ -11,8 +11,320 @@
+  <string name="fetch_xform_failed_unknown_reason">Une erreur inconnue est survenue.</string>
+  <string name="fetch_xform_failed_no_forms_found">Formulaire n\'a pas été trouvé sur le serveur.</string>
+  <string name="fetch_xform_failed_server_auth">Impossible d\'accéder à la forme sur le serveur. S\'il vous plaît confirmer que votre nom d\'utilisateur et mot de passe sont corrects.</string>
+  <string name="fetch_xform_failed_server_bad_endpoint">Erreur de récupération de la liste sous forme de serveur.</string>
+  <string name="fetch_xform_failed_server_failed_to_fetch">Impossible de récupérer le formulaire à partir du serveur.</string>
+  <string name="fetch_xform_failed_server_unknown">Erreur de récupération le formulaire à partir du serveur.</string>
+  <string name="retrieving_encounter_form_title">Récupération forme</string>
+  <string name="retrieving_encounter_form_message">S\'il vous plaît patienter pendant que la forme demandée est récupéré.</string>
+  <string name="syncing_patients">Synchronisation des patients…</string>
+  <string name="syncing_concepts">Synchronisation des concepts…</string>
+  <string name="syncing_charts">Synchronisation des graphiques…</string>
+  <string name="syncing_observations">Synchronisation observations…</string>
+  <string name="syncing_orders">Synchronisation des commandes…</string>
+  <string name="syncing_forms">Synchronisation des formes…</string>
+  <string name="syncing_locations">Synchronisation des emplacements…</string>
+  <string name="syncing_users">Synchronisation des utilisateurs…</string>
+  <string name="completing_sync">Fin de la synchronisation…</string>
+  <string name="sync_in_progress">Début de la synchronisation…</string>
+  <string name="submit_xform_failed_unknown_reason">Quelque chose a mal soumettre le formulaire.</string>
+  <string name="submit_xform_failed_server_auth">Erreur forme soumettant: n\'a pas pu authentifier avec le serveur.</string>
+  <string name="submit_xform_failed_server_timeout">Timed out tout en envoyant formulaire au serveur.</string>
+  <string name="submitting_encounter_form_title">Envoi de forme</string>
+  <string name="submitting_encounter_form_message">S\'il vous plaît patienter pendant que les données du formulaire sont soumis.</string>
+  <string name="consciousness">Conscience</string>
+  <string name="condition">État</string>
+  <string name="cancel_sync">Annuler synchronisation</string>
+  <string name="cancelling_sync">Annulation de la synchronisation…</string>
+  <string name="sync_starting">Synchronisation partir…</string>
+  <string name="troubleshoot_wifi_disabled">Wifi est désactivé</string>
+  <string name="troubleshoot_wifi_disabled_action_enable">Activer</string>
+  <string name="troubleshoot_wifi_disconnected">Wifi est déconnecté</string>
+  <string name="troubleshoot_wifi_disconnected_action_connect">Connectez</string>
+  <string name="troubleshoot_server_auth">Serveur nom d\'utilisateur / mot de passe peut être incorrect</string>
```

Comment:
not sure, but maybe 'Nom d\'utilisateur / mot de passe du serveur peut être incorrect'